%YAML 1.0yaml
---
height: 1024
width: 1280

K: !!opencv-matrix
   rows: 3
   cols: 3
   dt: f
   data: [2495.5,   0.,  664.,
          0.,    2504.5, 493.5,
          0.,      0.,    1.]
           
D: !!opencv-matrix
   rows: 1
   cols: 5
   dt: f
   data: [-0.3787, 0.11, 0., 0., 0.]
   
Lidar2Camera: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [0.0635, -0.9978, 0.0204, 0.0532, 
          0.0394, -0.0179, -0.9991, -0.501, 
          0.9972, 0.0642, 0.0382, -0.5131, 
           0,           0,           0,          1]
           
Base2Lidar: !!opencv-matrix
   rows: 4
   cols: 4
   dt: d
   data: [0.99875,   0.0499792, 0, -1.24444,
         -0.0499792, 0.99875,   0,  0.0422491,
          0,         0,         1, -1.51,
          0,         0,         0,  1]
